src/snipe/api/snipe_sig.py

```
@@ -1,10 +1,12 @@
 import heapq
 import logging
+
+import sourmash.save_load
 from snipe.api.enums import SigType
 from typing import Any, Dict, Iterator, List, Optional, Union
```

Ruff

`typing.Any` imported but unused

Remove unused import

(F401)